src/sql/magic.py

```diff
@@ -1,5 +1,6 @@
 import json
 import re
+from ipywidgets import interact
```

Comment:
let's make this an optional dependency

```suggestion
try:
    from ipywidgets import interact
except ModuleNotFoundError:
    interact = None
```

Reply:
Fix in [82184f6](https://github.com/ploomber/jupysql/pull/293/commits/82184f6bdc7731a290463ff834731efb4d1af651)